### DISCUSSION TITLE
Stripe Fees

### DISCUSSION THREAD

**Discussion by jeverington** [pretix/pretix#4128] [Ideas]
It would be great to have stripe fees come into the reports generated by Pretix. At present the reports are not as useful and we can't send them out to stakeholders of our events.

**Comment by jxs714** [comment]
> It would be great to have stripe fees come into the reports generated by Pretix. At present the reports are not as useful and we can't send them out to stakeholders of our events.

Hey guys, I second this as well. I am using Stripe + Service Fee plugin and add taxes to purchases and it would be very nice to have these things totaled into the final export instead of having to put more work on an accountant (or ourselves if we handle our own accounting) to get these numbers in. Here is a screenshot? Maybe I am missing something in the export? Screenshot attached
![pretix - accounting export](https://github.com/pretix/pretix/assets/4393073/74bc28dc-f8b9-4d82-a0e3-6ccf5a6e6f39)

**Comment by raphaelm** [comment]
We have considered this in the past but never built it since stripe is the *only* payment provider of the 25+ we support we could theoretically get that info, so it would be incomplete info in most setups

**Reply by jeverington** [reply]
Could it be added to all payment providers with a fallback of the user being able to enter basic fees (fixed and percentage).

**Reply by pc-coholic** [reply]
In theory: Sure. But I don't think we're gonna go there and do that, since this is just a recipe for desaster.

Just take PayPal as an example: Their fees vary depending on the monthly amount you are transacting with them. Or any of the hundreds of other payment providers that we support, where every type of transaction has a different cost: Interchange++ will do that for you...

Adding such a dumbed-down fixed fee to the report will cause confusion to people who'll take the report at face value.

If you would like to do this, you'll have to implement it yourself, possibly by subclassing the export and tacking on your desired changes. That way you can even have it in a plugin and not interfere with the update process.

**Reply by jeverington** [reply]
As per the example by [jxs714](https://github.com/jxs714) people are already trying to do this in many different hacky ways. Having additional ways to make the report more accurate surely is beneficial rather than having a report that is never going to be correct without the transaction fees on it.
